pwiz_tools/Skyline/SettingsUI/FormulaBox.cs

```diff
@@ -497,7 +498,7 @@ private void h3ToolStripMenuItem_Click(object sender, EventArgs e)
         private void helpToolStripMenuItem_Click(object sender, EventArgs e)
         {
             var helpText = FormulaHelpText;
-            MessageBox.Show(this, helpText, Resources.FormulaBox_helpToolStripMenuItem_Click_Formula_Help);
+            MultiButtonMsgDlg.Show(this, helpText, MessageBoxButtons.OK, Resources.FormulaBox_helpToolStripMenuItem_Click_Formula_Help);

```

Comment:
Again, custom title. I'll extend MessageDlg to support that.

Reply:
I would prefer not to have custom titles at all. That is the Microsoft Office standard I think we should follow. You would have to make a very strong case for why we need custom title text, not just pass along what got done with MessageBox simply because it offers flexibility we don't want to be used in our code.

Reply:
The current MessageDlg implementation intentionally implements the Microsoft Office standard without offering the option to do otherwise. I would prefer not to have examples for how to get around that in the code.